### PR TITLE
Line-breaking snippets at 121 characters to avoid having horizontal scrollbars

### PR DESCRIPTION
**Pull request by AlexJerabek**
I made a couple exceptions to this for large XML or image strings.

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/excel-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/excel-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/office-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/office-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/onenote-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/onenote-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/visio-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/visio-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/word-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/word-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.bindingcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.bindingcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartfill.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartfill.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartlineformat.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartlineformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartpointscollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartpointscollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.nameditem.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.nameditem.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheetprotection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.worksheetprotection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.asyncresult.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.asyncresult.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.binding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.binding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.bindings.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.bindings.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlnode.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.customxmlnode.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlpart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.customxmlpart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlparts.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.customxmlparts.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.document.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.document.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.tablebinding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.tablebinding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.notebook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.notebook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.notebookcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.notebookcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.page.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.page.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.paragraphcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.paragraphcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.richtext.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.richtext.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.section.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.section.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.sectiongroupcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.sectiongroupcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.recurrence.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/outlook/office.recurrence.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/visio/visio.hyperlinkcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/visio/visio.hyperlinkcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.body.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.body.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.contentcontrol.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.contentcontrol.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.contentcontrolcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.contentcontrolcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.paragraphcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.paragraphcollection.yml) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/scripts/package-lock.json](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/generate-docs/scripts/package-lock.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C22%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201808222121027024-134%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fYUPBHErPzHulDMB412%2FfEqsYLS3jp8C23RgaCPkA54%3D&st=2018-08-22T21%3A21%3A00Z&se=2018-09-22T21%3A26%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/code-snippets/word-snippets.yaml [hunk 1]
```diff
@@ -950,7 +956,9 @@ Word.ContentControlCollection.getByTitle:
             if (contentControlsWithTitle.items.length === 0) {
                 console.log("There isn't a content control with a title of 'Enter Customer Address Here' in this document.");
```
Comment (ElizabethSamuel-MSFT) [inline docs/code-snippets/word-snippets.yaml]: this seems longer than 121

File: docs/docs-ref-autogen/excel/excel.range.yml [hunk 2]
```diff
@@ -615,16 +611,14 @@ items:
...
 
-
           ```javascript
-
-          // Gets an object that represents the entire row of the range (for example, if the current range represents
-          cells "B4:E11", it's GetEntireRow is a range that represents rows "4:11").
-
+          // Gets an object that represents the entire row of the range 
+          // (for example, if the current range represents cells "B4:E11", 
+          // it's GetEntireRow is a range that represents rows "4:11").
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/excel/excel.range.yml]: nit: "it's" should be "its"

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/excel-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/excel-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/office-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/office-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/onenote-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/onenote-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/visio-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/visio-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/word-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/word-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.bindingcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.bindingcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartfill.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartfill.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartlineformat.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartlineformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartpointscollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartpointscollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.nameditem.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.nameditem.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheetprotection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.worksheetprotection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.asyncresult.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.asyncresult.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.binding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.binding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.bindings.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.bindings.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlnode.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.customxmlnode.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlpart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.customxmlpart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlparts.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.customxmlparts.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.document.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.document.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.tablebinding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.tablebinding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.notebook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.notebook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.notebookcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.notebookcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.page.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.page.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.paragraphcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.paragraphcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.richtext.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.richtext.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.section.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.section.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.sectiongroupcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.sectiongroupcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.recurrence.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/outlook/office.recurrence.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/visio/visio.hyperlinkcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/visio/visio.hyperlinkcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.body.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.body.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.contentcontrol.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.contentcontrol.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.contentcontrolcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.contentcontrolcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.paragraphcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.paragraphcollection.yml) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/scripts/package-lock.json](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/generate-docs/scripts/package-lock.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C22%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201808222133473268-134%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zNvE9VVyT3nCO%2FcVCd2XU88MJjA8gcdVGE2diFA%2F2w4%3D&st=2018-08-22T21%3A33%3A42Z&se=2018-09-22T21%3A38%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT. Should be fixed now.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/excel-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/excel-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/office-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/office-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/onenote-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/onenote-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/visio-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/visio-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/code-snippets/word-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/code-snippets/word-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.bindingcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.bindingcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartfill.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartfill.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartlineformat.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartlineformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartpointscollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.chartpointscollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.nameditem.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.nameditem.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheetprotection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/excel/excel.worksheetprotection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.asyncresult.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.asyncresult.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.binding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.binding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.bindings.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.bindings.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlnode.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.customxmlnode.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlpart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.customxmlpart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlparts.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.customxmlparts.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.document.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.document.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.tablebinding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/office/office.tablebinding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.notebook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.notebook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.notebookcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.notebookcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.page.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.page.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.paragraphcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.paragraphcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.richtext.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.richtext.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.section.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.section.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/onenote/onenote.sectiongroupcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/onenote/onenote.sectiongroupcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.recurrence.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/outlook/office.recurrence.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/visio/visio.hyperlinkcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/visio/visio.hyperlinkcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.body.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.body.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.contentcontrol.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.contentcontrol.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.contentcontrolcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.contentcontrolcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/word/word.paragraphcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/docs/docs-ref-autogen/word/word.paragraphcollection.yml) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/scripts/package-lock.json](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LineBreakSnippets/generate-docs/scripts/package-lock.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C22%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201808222152213314-134%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=eDZjrMVmhRCm2uSJk3gsrvqiZSYt08ne56cYZu8MCVQ%3D&st=2018-08-22T21%3A52%3A19Z&se=2018-09-22T21%3A57%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**